test/test_core.py

```diff
@@ -5609,6 +5609,10 @@ def test_statx(self):
     self.set_setting("FORCE_FILESYSTEM")
     self.do_runf('stat/test_statx.c', 'success')
 
+  def test_umask(self):
```

Comment:
Maybe put this alongside the existing `test_umask_0` in `test_other.py` (and put the test itself in `test/other` directory).

Reply:
I guess we should add `@also_with_wasmfs` to this test?

Reply:
Good idea. I'm also thinking of adding `@crossplatform`.

Reply:
Suggestion applied in https://github.com/emscripten-core/emscripten/pull/22589/commits/75827a0e4478490e9f4dc04699ac3af968537411.